lib/src/common/orgs_service.dart

```diff
@@ -167,6 +175,29 @@ class OrganizationsService extends Service {
     return completer.future;
   }
 
+  /// Returns the membership status for a user in a team given the [orgName] and [teamName].
```

Comment:
```suggestion
  /// Returns the membership status for [user] in [teamName] given the [orgName].
```

Reply:
I added your suggestion but directly since the code was outdated with a change.